db/models.py

```diff
@@ -14,3 +14,41 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(
+        to="Actor",
+        related_name="movies"
+    )
+    genres = models.ManyToManyField(
+        to="Genre",
+        related_name="genres"
```

Comment:
related_name="genres"  is a bit confusing. If you were to access this related name from a Genre object, it would suggest that you would get a QuerySet of Genre objects, which is not the case. What you actually get is a QuerySet of Movie objects that belong to that genre.

A more appropriate related_name for the genres field is "movies"